src/atlas/interpolation/method/cubedsphere/CubedSphereBilinear.cc

```diff
@@ -75,24 +75,21 @@ void CubedSphereBilinear::do_setup(const FunctionSpace& source, const FunctionSp
                     std::to_string(i) + ".");
             }
 
-            tileIndex.push_back(tijView(cell.idx, 0));
```

Comment:
tileIndex is not used elsewhere? dead code?

Reply:
It's used in our model interface, but I plan on removing the code that before we advance our current Atlas tag.